torch/csrc/profiler/python/init.cpp

```diff
@@ -292,6 +293,12 @@ void initPythonBindings(PyObject* module) {
   m.def(
       "_disable_execution_graph_observer",
       &torch::profiler::impl::disableExecutionGraphObserver);
+
+  py::class_<CapturedTraceback, std::shared_ptr<CapturedTraceback>>(
+      m, "CapturedTraceback");
+  m.def("gather_traceback", CapturedTraceback::gather);
+  m.def("symbolize_tracebacks", py_symbolize);
+  installCapturedTracebackPython();
```

Comment:
I'd definitely appreciate pyi type stubs for tehse bindings at the appropriate spot (maybe _C/__init__.pyi.in?)

Reply:
And docs!